contracts/libraries/ProxyDeltas.sol

```diff
@@ -15,68 +15,58 @@ struct ProxyDeltas {
 /// The list works optimally if after applying a series of changes it's iterated over.
 /// The list uses 2 words of storage per stored cycle.
 library ProxyDeltasImpl {
+    using ProxyDeltasImpl for ProxyDeltas;
+
     struct ProxyDeltaStored {
         uint64 next;
         int128 thisCycleDelta;
+        uint64 isAttached;
         // --- SLOT BOUNDARY
         int128 nextCycleDelta;
+        uint128 slotFiller;
```

Comment:
Can you explain why this can’t just be omitted?

Reply:
Added a comment. The unused parts of the slot are assumed to be out of control and worth preserving. When a non-full slot is deleted, Solidity will read it, zero out the fields it manages and write back whatever is left. AFAIK it's supposed to let you do some magic with the empty parts from Yul and manage them yourself. Personally I'm not convinced that it's a useful design decision, for 99 % of contracts it's a nasty trap, but that's the way it is.